txpipe/input_cats.py

```diff
@@ -332,7 +339,7 @@ def write_output(self, start, target_size, photo_cols, metacal_cols, photo_file,
         n = len(photo_data['id'])
         end = min(start + n, target_size)
 
-        assert photo_data['id'].min()>0
```

Comment:
I think in CosmoDC2 negative IDs indicated something bad, but that's clearly not important here.